addons/cert.py

```diff
@@ -311,6 +310,7 @@ def msc24(data):
         exp46(cfg)
         int31(cfg, data.platform)
         str05(cfg)
+        str07(cfg)
```

Comment:
this `str07` does not seem to belong in this PR.

Reply:
It's just the merged version from your master branch...

Reply:
sorry! my fault..